BookPlayer/Library/ItemList Screen/ItemListViewModel.swift

```diff
@@ -1311,6 +1311,43 @@ extension ItemListViewModel {
       )
     ))
   }
+
+  func getDownloadURL(for givenString : String) -> URL? {
+    let givenUrl = URL(string: givenString)
+
+    if let givenUrl, let hostname = givenUrl.host {
+      if hostname == "drive.google.com" {
+        return getGoogleDriveURL(for: givenUrl)
+      } else if hostname == "www.dropbox.com" {
+        return getDropboxURL(for: givenUrl)
+      }
+    }
+    return nil
+  }
+
+  func getGoogleDriveURL(for url: URL) -> URL?
+  {
```

Comment:
minor nit, can we keep the opening brace on the same line as the function declaration?